### PR TITLE
Add timeout setting support

### PR DESCRIPTION
**Pull request by invidian**

### CODE REVIEW

File: manifests/init.pp [hunk 1]
```diff
@@ -29,6 +35,8 @@
 class sslh(
   $package_version = 'present',
   $listen_address  = '0.0.0.0:443',
+  $timeout         = 2,
+  $on_timeout      = '127.0.0.1:22',
```
Comment (nhinds) [inline manifests/init.pp]: This doesn't seem to match the default from the sslh documentation ("ssh").

**Comment by nhinds** [comment]
Instead of copying defaults into this module, can you default the parameters to `undef` and conditionally add the arguments to the config file? That way if the parameters  aren't explicitly specified, they don't get passed to sslh

**Comment by invidian** [comment]
Sure, that sounds reasonable. I didn't do that, because I didn't want to make it too complicated.

**Comment by invidian** [comment]
Updated. It doesn't look very good, but it's working properly.

**Comment by nhinds** [comment]
Using fragments or adding newlines would make it look nicer (there are already newlines in the options thanks to the other fragments), but I'm not overly fussed. LGTM.

**Pull request merged**